stl/inc/span

```diff
@@ -346,7 +348,11 @@ inline constexpr bool _Is_span_compatible_range = conjunction_v<
 // [views.span]
 // CLASS TEMPLATE span
 template <class _Ty, size_t _Extent = dynamic_extent>
-class span : public _Span_extent_type<_Extent> {
+class span : private _Span_extent_type<_Ty, _Extent> {
+    using _Mybase = _Span_extent_type<_Ty, _Extent>;

```

Comment:
Style: We generally say `private:` at the top of a `class`, despite that being the default. (But we don't say `public:` for `struct`.) Perhaps we should change that convention - it is a relatively rare exception to "don't say redundant things" - but that's the current state.

(`private` is also the default for `class` inheritance but almost nobody realizes that, so repeating the keyword there is strongly justified in any event.)

Reply:
Don